listenbrainz/webserver/static/js/src/personal-recommendations/PersonalRecommendationsModal.tsx

```diff
@@ -0,0 +1,258 @@
+        users,
+        blurb_content: blurbContent,
+      };
+      try {
+        const status = await APIService.submitPersonalRecommendation(
+          currentUser.auth_token,
+          currentUser.name,
+          metadata
+        );
+        if (status === 200) {
+          newAlert(
+            "success",
+            `You recommended this track to ${users.length} user${
+              users.length > 1 ? "s" : ""
+            }`,
+            `${metadata.artist_name} - ${metadata.track_name}`
+          );
+          this.setState({ blurbContent: "" });
+        }
+      } catch (error) {
+        this.handleError(error, "Error while recommending a track");
+      }
+    }
+  };
+
+  render() {
+    const { recordingToPersonallyRecommend } = this.props;
+    if (!recordingToPersonallyRecommend) {
+      return null;
+    }
+    const { blurbContent, users, suggestions } = this.state;
+    const {
+      track_name,
+      artist_name,
+    } = recordingToPersonallyRecommend.track_metadata;
+    const { APIService, currentUser } = this.context;
+    return (
+      <div
+        className="modal fade"
+        id="PersonalRecommendationModal"
+        tabIndex={-1}
+        role="dialog"
+        aria-labelledby="PersonalRecommendationModalLabel"
+        data-backdrop="static"
+      >
+        <div className="modal-dialog" role="document">
+          <form className="modal-content">
+            <div className="modal-header">
+              <button
+                type="button"
+                className="close"
+                data-dismiss="modal"
+                aria-label="Close"
+                onClick={this.closeModal}
+              >
+                <span aria-hidden="true">&times;</span>
+              </button>
+              <h4 className="modal-title" id="PersonalRecommendationModalLabel">
+                Personally recommend this recording
+              </h4>
+            </div>
+            <div className="modal-body">
+              {users.map((user) => {
+                return (
+                  <NamePill
+                    title={user}
+                    // eslint-disable-next-line react/jsx-no-bind
+                    closeAction={this.removeUser.bind(this, user)}
+                  />
+                );
+              })}
+              <input
+                type="text"
+                className="form-control"
+                onChange={this.searchUsers}
+                placeholder="Search and add users"
+              />
+              <SearchDropDown
+                suggestions={suggestions}
+                // eslint-disable-next-line react/jsx-no-bind
+                action={this.addUser}
+              />
+              <p>
+                Tell the users selected above why are you recommending them{" "}
+                <b>
+                  {" "}
+                  {track_name} by {artist_name}
+                </b>
+                (Optional)
```

Comment:
Making this smaller and gray text, as well as adding a missing space:
```suggestion
                <small> (Optional)</small>
```
This also requires changing some css used for the character count: currently the rule is applied to all `<small>` elements (`#PersonalRecommendationModal small`), so we probably want to add a className `character-count` (for example) to the corresponding element and target that instead (`#PersonalRecommendationModal .character-count`

Before 
![image](https://user-images.githubusercontent.com/6179856/196186981-d7304e0a-e8dc-4667-990c-cb4bf62c843f.png)

After:
![image](https://user-images.githubusercontent.com/6179856/196187322-be3ce1ca-726c-4171-b838-ea16771d5985.png)